app/src/main/java/com/findmyrecycling/MainActivity.kt

```diff
@@ -95,9 +141,28 @@ class MainActivity : ComponentActivity() {
 
     @Composable
     fun ProfileMenu(){
-        Box(Modifier.fillMaxWidth(), contentAlignment = Alignment.CenterEnd) {
-            Icon(imageVector = Icons.Filled.Menu, contentDescription = "")
+        val context = LocalContext.current
+        Box(
+            modifier = Modifier
+                .background(color = Color.LightGray)
+                .fillMaxWidth()
+                .padding(5.dp),
+                contentAlignment = Alignment.CenterStart)
+
+        {
+            Row()
+            {
+                IconButton(
+                    onClick = { Toast.makeText(context, "Menu Clicked", Toast.LENGTH_LONG).show() }
+                ) {
+                    Icon(imageVector = Icons.Filled.Menu, contentDescription = "")
+                }
 
+                Text("Find My Recycling",
+                    textAlign = TextAlign.Center,
+                    fontSize = 23.sp,
+                    modifier = Modifier.padding(top = 5.dp))
+            }
```

Comment:
This is completely optional, but the app looked very plain without a title bar, so I added you're app's name to the blank space in the "Box" that also contained the menu iconButton.   I noticed that this was included in the storyboard design, so assuming it's still part of the overall design plan